src/util.cpp

```diff
@@ -196,8 +197,14 @@ std::string AbsolutePath(const std::string &filepath) {
       char abs_path[MAX_PATH];
       return GetFullPathNameA(filepath.c_str(), MAX_PATH, abs_path, nullptr)
     #else
-      char abs_path[PATH_MAX];
-      return realpath(filepath.c_str(), abs_path)
+	  char *abs_path_temp = realpath(filepath.c_str(), nullptr);
```

Comment:
this is still weirdly indented..

Reply:
Should be better! Now to figure out why 32bit arm build seems to fail.